duckduckgo/Android
# Navigation

Reviewer: @subsymbolic 
Asana: https://app.asana.com/0/0/361925093999571

**Description**: Implement Navigation Bar


**Steps to test this PR**:
1. Open the app and notice the bottom navigation bar
2. Confirm that the navigation bar icons and the omnibar menu are consistent with the Zeplin design


###### Reviewer Checklist:


###### PR DRI Checklist:
